common/survey/categoryItem.js

```diff
@@ -12,31 +12,23 @@ const keys = {
 
 const props = {
   code: 'code',
-  labels: 'labels'
+  descriptions: 'descriptions',
+  extra: 'extra',
+  labels: 'labels',
 }
```

Comment:
are you using the keys labels and descriptions? if yes, then we should use ObjectUtils, what do you think?

Reply:
yes, I'll remove them and use ObjectUtils instead